docs/ref/contrib/gis/tutorial.txt

```diff
@@ -53,12 +53,9 @@ Create a Spatial Database
 
 First, create a spatial database for your project.
```

Comment:
Right, what I meant was, the note at the beginning of the section says, "MySQL and Oracle users can skip this section because spatial types are already built into the database." This is now the case with PostgreSQL -- there's no special setup required, so do we need to mention PostgreSQL?

I was thinking something like:
```
If you are using SQLite and SpatiaLite, consult the instructions on how to
create a :ref:`SpatiaLite database <create_spatialite_db>`. Otherwise, no
special setup is required and you can create a database as you would for any
other project.
```
Possibly it's also worth mentioning that the PostgreSQL user must be able to run `CREATE EXTENSION`?